ejb3/src/main/java/org/jboss/as/ejb3/component/singleton/SingletonComponentDescription.java

```diff
@@ -125,7 +125,9 @@ public void configure(final DeploymentPhaseContext context, final ComponentDescr
                     if(interceptorConfig.getPostConstruct() != null) {
```

Comment:
The TCK is for testing compliance and not intended as a 100% code (or spec) coverage QE tool.  This is especially true for EE technology changes that are added late in an EE specification cycle.  I'm not trying to make excuses for the TCK but help explain the general thinking.  Please continue to add tests to the WildFly testsuite to increase our coverage.